float64 omega_l
float64 omega_r

---
float64 vel_l
float64 vel_r
